index.js

```diff
@@ -66,7 +68,7 @@ var S3Sync = klass(function (config, options) {
         this.writeDigestFile(function (err, resp) {
           debug('wrote digest file')
           this.options.complete && this.options.complete()
-        })
+        }.bind(this))
```

Comment:
Line 98 you mean? I thought so too, but without the bind on Line 71 it wasn't working.